docs/docs/overview.md

```diff
@@ -5,39 +5,47 @@ weight: 11
 
 ## ResyncPeriod
 
-Grafana doesn't have any webhooks or similar ways of giving information to the operator that a Grafana resource, like a dashboard, has changed.
-Due to this the Grafana operator has to constantly poll the Grafana API to test for changes in the dashboard.
+Grafana doesn't have any webhooks or similar ways of notifying the operator that a Grafana resource, like a dashboard, has changed.
+Due to this the Grafana operator constantly polls the Grafana API to test for changes and overwrite the resources, reconciling towards the desired state.
+
+We describe this loop as a synchronizing resources with Grafana instances.
 
 To control how often this polling should occur, you can set the `spec.resyncPeriod` field.
-This field tells the operator how often it should poll the Grafana instance for changes.
+This field tells sets how often it should poll the Grafana instance for changes for the specific resource.
+
+If a dashboard has changed, the operator will overwrite and synchronize the dashboard after `10m` per default.
+
+This can of course be annoying for developers actively updating a resource. The recommended workflow is to duplicate the dashboard/alert/other and work on the copy.
```

Comment:
Should we mark the PR as fixing #1597 then?

Reply:
I think that line was present before my edits: https://github.com/grafana/grafana-operator/blob/129224eef2ae5ab09bb95a868a9cdc2e28013809/docs/docs/overview.md?plain=1#L35-L36

So it's not really fixed in this PR but rather in: https://github.com/grafana/grafana-operator/pull/1466